### PR TITLE
Fix autocomplete search

### LINKED ISSUE #1223: Fix autocomplete search in groups and zones tab
**Describe the bug**
- The autocomplete search currently has improper styling. 
- Also, there's a minor bug in zone autocomplete search. When search by admin group is selected and then unselected, the autocomplete still suggests groups instead of zones.

Screenshot 2023-02-24 at 5 48 30 PM

### PR DESCRIPTION
**Pull request by Aravindh-Raju**
Fixes #1223.

Changes in this pull request:
- Fixes the styling of autocomplete search in zones and groups tab. Also, fixes a minor bug in zone autocomplete search.

### CODE REVIEW

File: modules/portal/public/lib/controllers/controller.zones.js [hunk 1]
```diff
@@ -157,9 +148,46 @@ angular.module('controller.zones', [])
                 $(this).removeClass("ui-corner-top").addClass("ui-corner-all");
               }
             });
+        } else {
+            $(".zone-search-text").autocomplete({
```
Comment (arpit4ever) [inline modules/portal/public/lib/controllers/controller.zones.js:152]: Can we avoid duplication of this section of the code ?

Comment (Aravindh-Raju) [reply modules/portal/public/lib/controllers/controller.zones.js]: Yes, made it a function and reused it.

**Review by nspadaccino — APPROVED** [review]
Looks good to me, styling looks good as well as the admin group search functionality

**Review by arpit4ever — APPROVED** [review]
LGTM

**Pull request merged**